plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeColumnHandle.java

```diff
@@ -151,7 +161,7 @@ public HiveColumnHandle toHiveColumnHandle()
     {
         return new HiveColumnHandle(
                 physicalName, // this name is used for accessing Parquet files, so it should be physical name
-                0, // hiveColumnIndex; we provide fake value because we always find columns by name
+                fieldId.orElse(0), // hiveColumnIndex; we provide fake value when delta.columnMapping.mode table property isn't id
```

Comment:
I am confused here: Is `fieldId` the same as  hive column index?

Reply:
No, but it's used to provide the position in `HiveColumnHandle`.